TASK-5/TASK-5/Menu.cs

```diff
@@ -0,0 +1,176 @@
+                    break;
+                case 4:
+                    AddItemToBag(users);
+                    break;
+                case 5:
+                    RemoveItemFromBag(users);
+                    break;
+                default:
+                    exit = false;
+                    break;
+            }
+        }
+    }
+
+    public static List<User> GenerateFiveUsers()
+    {
+        var users = new List<User>();
+        for (int i = 0; i < 5; i++)
+        {
+            users.Add(UserGenerator.GenerateUserWithItems());
+        }
+        
+        return users;
+    }
+
+    private static void DisplayEveryClient(List<User> usersContainer)
+    {
+        Console.WriteLine();
+        foreach (var user in usersContainer)
+        {
+            Console.WriteLine($"{user.FullName} || Age is {user.Age} || ID is {user.PassportId}");
+        }
+    }
+
+    private static void DisplayClientPurchases(List<User> users, int userNumber)        //counting from 1;
+    {
+        decimal totalPrice = 0;
+        foreach (var items in users[userNumber-1].GroceryBag)
+        {
+            Console.WriteLine($"{items.Barcode} || {items.Category} || {items.Price} || {items.ItemName}");
+            totalPrice += items.Price;
+        }
+        Console.WriteLine($"Total: {totalPrice}$");
+    }
+    
+    private static void DisplayClientPurchases(List<User> users)            //overload with manual number input
+    {
+        decimal totalPrice = 0;
+        string input;
+        int itemNumber;
+        do
+        {
+            Console.WriteLine("Please, enter User number (starts from 1): ");
+            input = Console.ReadLine();
+        } while (!int.TryParse(input, out itemNumber));
+        
+        foreach (var items in users[itemNumber-1].GroceryBag)
+        {
+            Console.WriteLine($"{items.Barcode} || {items.Category} || {items.Price} || {items.ItemName}");
+            totalPrice += items.Price;
+        }
+        Console.WriteLine($"Total: {totalPrice}$");
+    }
+   
+    private static void DisplayItems(List<Item> itemBag)     //why am i not using this???
+    {
+        Console.WriteLine("\n");
+        foreach (var item in itemBag)
+        { 
+            Console.WriteLine($"ItemName {item.Barcode} || {item.Category}" +
+                              $" || Barcode #{item.ItemName} || costs {item.Price}$");
+        }
+    }
+    
+    private static void GenerateUserFromConsole(List<User> users) 
+    {
+        Console.Write("You are going to create new user \n Enter new user's passportID \n");
+        var inputPassportId = Convert.ToInt32(Console.ReadLine());
+
+        if (Validation.SameIdChecker(inputPassportId, users))
+        {
+            users.Add(UserGenerator.GenerateUserFromConsole(inputPassportId));
+            Console.WriteLine("You've added new user successfully");
+        }
+        else
+        {
+            Console.WriteLine("User cannot be added");
+        }
+    }
```

Comment:
Why do you have similar names for methods?
Menu.GenerateUserFromConsole(inputPassportId)
UserGenerator.GenerateUserFromConsole(inputPassportId)

You can name it as CreateUser